middleware/errorHandler.js

```diff
@@ -23,13 +23,13 @@ function errorHandler(err, req, res, next) {
       },
     })
   }
-  if (err.name === "PayloadTooLargeError") {
-    // Error thrown by large payload is done by express
-    logger.info(errMsg)
-    return res.status(413).json({
+  // Error handling for non-Isomer response errors
+  if (err.name && err.message && err.status) {
+    logger.info(`Internal server error: ${errMsg}`)
+    return res.status(err.status).json({
       error: {
         name: err.name,
-        code: 413,
+        code: err.status,
         message: err.message,
       },
     })
```

Comment:
This seems to include Github errors as well, which we should be monitoring - doing this will bypass our monitoring and possibly cause us to miss errors that we should be handling. I think we should log the actual error and status code, but continue to send a 500 error to the frontend, and send the logger message `Unrecognized internal server error: ${errMsg}`, as this is the string we are looking out for.

Reply:
I will reverted to the previous version in my revised PR. This isn't important enough to change right now, and it sounds like we should think it through some more.